private/buf/cmd/buf/command/curl/curl.go

```diff
@@ -197,7 +197,7 @@ exit code that is the gRPC code, shifted three bits to the left.
 
 type flags struct {
 	// Flags for defining input schema
-	Schema string
+	Schema []string
```

Comment:
nit: I'd rename this to `Schemas` (but keep the flag name the same)

Reply:
done